lib/tests/streamlit/elements/layouts_test.py

```diff
@@ -112,6 +114,7 @@ def test_not_equal_width_float_columns(self):
 
         for column in columns:
             with column:
+                """Noop"""
```

Comment:
nit (here and throughout this test): could this be a normal comment rather than a docstring?

Reply:
yep, the linter seems to accept that! I have updated it